backend/core/src/email/email.service.ts

```diff
@@ -329,6 +328,22 @@ export class EmailService {
     return partials
   }
 
+  private async govSend(rawHtml: string, subject: string) {
+    // juice inlines css to allow for email styling
+    const inlineHtml = juice(rawHtml)
+    const govEmailXml = `<bulletin>\n <subject>${subject}</subject>\n  <body><![CDATA[\n     
+      ${inlineHtml}\n   ]]></body>\n   <sms_body nil='true'></sms_body>\n   <publish_rss type='boolean'>false</publish_rss>\n   <open_tracking type='boolean'>true</open_tracking>\n   <click_tracking type='boolean'>true</click_tracking>\n   <share_content_enabled type='boolean'>true</share_content_enabled>\n   <topics type='array'>\n     <topic>\n       <code>doorway-listings</code>\n     </topic>\n   </topics>\n   <categories type='array' />\n </bulletin>`
```

Comment:
suggestion: Can you also make "doorway-listings" an environment variable? That is the topic name and it could be different in stage vs prod

Reply:
Done!